Sources/EventOffline.swift

```diff
@@ -63,7 +63,11 @@ extension SentryClient {
 	/// Path of directory to which events will be saved in offline mode
 	private func directory() -> String? {
 		guard let documentsPath = NSSearchPathForDirectoriesInDomains(.DocumentDirectory, .UserDomainMask, true).first else { return nil }
-		let directory: String = "\(directoryNamePrefix)\(dsn.serverURL.absoluteString.hashValue)"
+        #if swift(>=2.3)
+		let directory: String = "\(directoryNamePrefix)\(dsn.serverURL.absoluteString!.hashValue)"
```

Comment:
Not too shabby haha. I don't know if I'm a huge fan of the force unwrap so I might see if there is something else we can default too even though that _should_ always be there